src/vs/workbench/parts/webview/electron-browser/webview-pre.js

```diff
@@ -258,6 +258,16 @@
 				color: var(--vscode-textPreformat-foreground);
 			}
 
+			code span.hljs-addition {
+				background: var(--vscode-diffEditor-insertedTextBackground, rgba(155, 185, 85, 0.2));
```

Comment:
This default color was picked up from here: https://github.com/Microsoft/vscode/blob/d9add550e9315d133484f5497362a830a71da29a/src/vs/platform/theme/common/colorRegistry.ts#L292